.ci/metricbeat/Makefile

```
@@ -0,0 +1,58 @@
+.PHONY: help
+.DEFAULT_GOAL := help
+BATS_VERSION = c706d1470dd1376687776bbe985ac22d09780327 #v1.1.0
+BATS_ASSERT = 9f88b4207da750093baabc4e3f41bf68f0dd3630 #v0.3.0
+BATS_SUPPORT = 004e707638eedd62e0481e8cdc9223ad471f12ee #v0.3.0
+LTS_ALPINE ?= 12-alpine
+
+help: ## Display this help text
+	@grep -E '^[a-zA-Z_-]+[%]?:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
+
+bats: ## Install bats in the project itself
+	@echo "Cloning bats-core"
+	@[ -d $(CURDIR)/bats-core ] \
+		|| git clone -n https://github.com/bats-core/bats-core.git $(CURDIR)/bats-core
+	@cd $(CURDIR)/bats-core \
+		&& git fetch --all \
+		&& git checkout -B v1.1.0 ${BATS_VERSION}
+
+bats-assert: ##Install bats-assert in the project itself
+	@echo "Clonning bats-assert"
+	@[ -d $(CURDIR)/tests/test_helper/bats-assert ] \
+		|| git clone -n https://github.com/ztombol/bats-assert $(CURDIR)/tests/test_helper/bats-assert
+	@cd $(CURDIR)/tests/test_helper/bats-assert \
+		&& git fetch --all \
+		&& git checkout -B v0.3.0 ${BATS_ASSERT}
+
+bats-support: ##Install bats-support in the project itself
+	@echo "Clonning bats-support"
+	@[ -d $(CURDIR)/tests/test_helper/bats-support ] \
+		|| git clone -n https://github.com/ztombol/bats-support $(CURDIR)/tests/test_helper/bats-support
+	@cd $(CURDIR)/tests/test_helper/bats-support \
+		&& git fetch --all \
+		&& git checkout -B v0.3.0 ${BATS_SUPPORT}
+
+prepare-test: bats bats-assert bats-support## Prepare the bats dependencies
+	@echo "Pulling Alpine image"
+	@docker pull node:${LTS_ALPINE}
```

Does the`read_version` function return the downloading string if the docker image has not been cached yet? If so, then something like the below snippet might help to cache it beforehand:
```suggestion
	@docker pull node:${LTS_ALPINE}
	@docker pull mikefarah/yq:2.4.0
```

---

Yeah, I tried that, and it pulls the image when running it if it's not present. But I'm going to add the image to this pull stage, so that it's clear and explicit the dependency with the image.

Thanks!